CHANGELOG.md

```diff
@@ -1,3 +1,449 @@
+    different clusters.
+    ([#710](https://github.com/kinvolk/lokomotive/pull/710)).
+
+#### Docs
+
+- Add `Type` column to Attribute reference table in [configuration
+ references](https://github.com/kinvolk/lokomotive/blob/v0.3.0/docs/configuration-reference)
+ ([#651](https://github.com/kinvolk/lokomotive/pull/651)).
+
+- Update contour configuration reference for usage with AWS
+   ([#674](https://github.com/kinvolk/lokomotive/pull/674)).
+
+- Add documentation related to the usage of `clc_snippets` for Packet and AWS
+    ([#657](https://github.com/kinvolk/lokomotive/pull/657)).
+
+- Improve documentation on using remote
+    [backends](https://github.com/kinvolk/lokomotive/blob/v0.3.0/docs/configuration-reference/backend/s3.md)
+    ([#670](https://github.com/kinvolk/lokomotive/pull/670)).
+
+- How to guide for setting up monitoring on Lokomotive
+    ([#480](https://github.com/kinvolk/lokomotive/pull/480)).
+
+- Add `codespell` section in [development
+    documentation](https://github.com/kinvolk/lokomotive/blob/v0.3.0/docs/development/README.md)
+    ([#700](https://github.com/kinvolk/lokomotive/pull/700)).
+
+- Include a demo GIF in the readme
+    ([#636](https://github.com/kinvolk/lokomotive/pull/636)).
+
+#### Bugfixes
+
+- Remove contour ingress workaround (due to an [upstream issue](https://github.com/projectcontour/contour/issues/403))
+  for ExternalDNS ([#635](https://github.com/kinvolk/lokomotive/pull/635)).
+
+#### Development
+
+- Do not show Helm release values in terraform output
+    ([#627](https://github.com/kinvolk/lokomotive/pull/627)).
+
+- Remove Terraform provider aliases from platforms code
+    ([#617](https://github.com/kinvolk/lokomotive/pull/617)).
+
+
+#### Miscellaneous
+
+- Following flatcar-linux/Flatcar#123, Flatcar 2513.1.0 for ARM contains the dig
+    binary so the workaround is no longer needed
+    ([#703](https://github.com/kinvolk/lokomotive/pull/703)).
+
+- Improve error message for `wait-for-dns` output
+    ([#735](https://github.com/kinvolk/lokomotive/pull/735)).
+
+- Add `codespell` to enable spell check on all PRs ([#661](https://github.com/kinvolk/lokomotive/pull/661)).
+
+### Upgrading from v0.2.1
+
+#### Configuration syntax changes
+
+There have been some minor changes to the configurations of following components:
+* contour
+* prometheus-operator.
+
+Please make sure new the configuration structure is in place before the upgrade.
+
+##### Contour component
+
+Optional `ingress_hosts` variable is now removed.
+
+old:
+
+```hcl
+component "contour" {
+  .
+  .
+  ingress_hosts = ["*.example.lokomotive-k8s.net"]
+}
+```
+
+new:
+
+```hcl
+component "contour" {
+  .
+  .
+}
+
+```
+
+##### Prometheus-operator component
+
+* Prometheus specific variables are now under a `prometheus` block.
+* A new optional `prometheus.ingress` sub-block is introduced to expose
+Prometheus over ingress.
+* Variable `external_url` is now removed and now configured under
```

Comment:
As `variable` is:
```hcl
variable "foo" {}
```

```suggestion
* Attribute `external_url` is now removed and now configured under
```

Attribute, because this is how HCL refers to it: https://github.com/hashicorp/hcl/tree/hcl2. (BTW Terraform is inconsistent and calls it `arguments` :facepalm: https://www.terraform.io/docs/configuration/syntax.html)